src/datachain/lib/dc.py

```diff
@@ -1887,6 +1889,7 @@ def to_parquet(
         path: Union[str, os.PathLike[str], BinaryIO],
         partition_cols: Optional[Sequence[str]] = None,
         chunk_size: int = DEFAULT_PARQUET_CHUNK_SIZE,
+        fs_kwargs: Optional[dict[str, Any]] = None,
```

Comment:
I have a concern here that we don't use `fs_kwargs` in all other places (e.g. anon=True, or `from_parquet` I think reads from a file object - file.get_fs() or something). can we do a bit of research on that end and unify or get rid of this additional kwargs?

Reply:
These kwargs are optional and are combined with the Catalog's `client_config` for any unified use cases, such as configuration that applies to read and write. I added this optional kwargs parameter here to be used if users need to specify a write-only custom configuration, such as an access token to be used on write, etc. that is not (or may not apply) on read or for the whole application / chain. For example, a token can be specified for Hugging Face filesystems on write, as described here: https://huggingface.co/docs/huggingface_hub/en/guides/hf_file_system#authentication but users may only want to specify this token on write to Hugging Face, not for other clouds or on read. I can rename this or change as desired, but it seems like having extra write-only kwargs can be useful in some cases.

Reply:
kk. It seems to me that it should be symmetrical (people might need to provide something extra on reads as well eventually) and then the question is - will we be able to do the same easy and w/o changing some logic in those methods (like from_parquet, etc).

should we update the docs here btw?

Reply:
Yes, the docs should be updated - I updated them in the latest commit. And shared (read and write) kwargs can be provided in `client_config` on Session or Catalog (as well as in environment variables) and these configuration settings will automatically be used for read and write. This `fs_kwargs` option just provides a way to provide write-only configuration, or override the shared configuration, only if necessary.